### PR TITLE
bump component library

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -249,7 +249,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/component-library": "^3.3.4",
+    "@department-of-veterans-affairs/component-library": "^3.5.0",
```
Comment (jerekshoe) [inline package.json:252]: Should we just bump this to ^3.6.0? That’s what it looks like is being installed anyways

**Review by jerekshoe — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (tmitchellgcio) [reply package.json]: yep just did that

**Comment by timwright12** [comment]
adding @k80bowman and @bkjohnson here for review

**Comment by tmitchellgcio** [comment]
> adding @k80bowman and @bkjohnson here for review

Thanks. I never know who all to add for some of these

**Review by grossb1 — APPROVED** [review]
LGTM

**Comment by tmitchellgcio** [comment]
> adding @k80bowman and @bkjohnson here for review

Anyone else I can contact to get this moving?

**Review by k80bowman — APPROVED** [review]
Sorry about the delay, needed to check on something.

**Review by theodur — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**